recipes/protobuf/all/test_package/conanfile.py

```diff
@@ -0,0 +1,27 @@
+import os
+import shutil
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        if os.path.exists(os.path.join(self.source_folder, "addressbook.pb.cc")):
+            os.remove(os.path.join(self.source_folder, "addressbook.pb.cc"))
+        if os.path.exists(os.path.join(self.source_folder, "addressbook.pb.h")):
+            os.remove(os.path.join(self.source_folder, "addressbook.pb.h"))
```

Comment:
Is this needed?

Reply:
I didn't understand it also.

Reply:
This is to delete existing generated code, which can force compiler to re-compile these source files. Actually I can barely remember the problem leading to this fix.